### ISSUE TITLE
import state by state emissions data

### ISSUE THREAD

**Issue by derekeder** [chihacknight/state-by-state-decarbonization#2]
Using the data prepped by @swatland610, lets import the US emissions data by state into a format that Gatsby and Recharts can ingest.

We'll base the display of these charts on the stacked bar charts of emissions Sean put together. Here's Illinois for reference:

![Screen Shot 2022-02-09 at 8 57 53 AM](https://user-images.githubusercontent.com/919583/153227332-4d904194-5532-4f0d-8d83-b10b29297d55.png)

This codebase uses [recharts](https://recharts.org) so we can base ours off of this stacked bar chart example: https://recharts.org/en-US/examples/StackedBarChart

Data output into `data/final/emissions.json` should be formatted roughly in this way:

```json
{
  "illinois": [
    {
      "year": "2001",
      "dirty_power": 1000000,
      "buildings": 1000000,
      "transportation": 1000000,
      "dumps, farms, industrial": 1000000
    },
    {
      "year": "2002",
      "dirty_power": 1000000,
      "buildings": 1000000,
      "transportation": 1000000,
      "dumps, farms, industrial": 1000000
    },
    {
      "year": "2003",
      "dirty_power": 1000000,
      "buildings": 1000000,
      "transportation": 1000000,
      "dumps, farms, industrial": 1000000
    },
    ...
  ],
  "indiana": [
    {
      "year": "2001",
      "dirty_power": 1000000,
      "buildings": 1000000,
      "transportation": 1000000,
      "dumps, farms, industrial": 1000000
    },
    ...
  ],
  ...
}

```

**Comment by swatland610** [comment]
hey @derekeder! I should be able to refactor the helper functions to output the data like this. I'll take a look in the next couple of days!

**Comment by derekeder** [comment]
closed via #1
